package.json

```diff
@@ -48,15 +48,15 @@
     "aws-sdk": "2.1692.0",
     "commitizen": "4.3.1",
     "concurrently": "9.2.0",
-    "cspell": "8.19.4",
+    "cspell": "9.2.0",
```

Comment:
Drops support for node 18

https://github.com/streetsidesoftware/cspell/blob/main/CHANGELOG.md#v900-2025-05-05